### ISSUE TITLE
Output reproducibility issue while converting Mvitv2 torch model to ONNX format

### ISSUE THREAD

**Issue by Kishor33174** [facebookresearch/SlowFast#695]
We tried to convert Mvitv2 torch model to onnx using torch.onnx.export.Even thought the input is same , we are getting 2 different outputs for torch and onnx models (which are way different). However on the dummy input we used while conversion its producing same output  for torch and onnx . How can convert our Mvitv2 torch model to onnx such that the same results are reproduced ?

**Comment by alpargun** [comment]
Can you share the code snippet you converted the PyTorch model to ONNX? But without your code, I assume:

- with torch.onnx.export(), did you set `export_params=True`?
- torch.onnx "does not handle nuances between training and eval mode" ([link](https://pytorch.org/docs/stable/onnx.html)), did you make sure the model is in eval mode, so that you don't see BatchNorm, Dropout, etc. behaviour?
